app/src/main/java/com/hedvig/app/feature/loggedin/ui/LoggedInViewModel.kt

```diff
@@ -35,9 +40,14 @@ abstract class LoggedInViewModel : ViewModel() {
 }
 
 class LoggedInViewModelImpl(
-    private val loggedInRepository: LoggedInRepository
+    private val loggedInRepository: LoggedInRepository,
+    chatClosedTracker: ChatEventStore
 ) : LoggedInViewModel() {
 
+    override val shouldOpenReviewDialog = chatClosedTracker.observeChatClosedCounter()
```

Comment:
Thinking if we should reset this counter somewhere? Perhaps after a new version?

Reply:
It's probably fine as is, for now. Could be wise to reset it on logging out though, I think!